eng/doc/fips/README.md

```diff
@@ -343,7 +349,7 @@ Not all OpenSSL versions are supported. OpenSSL does not maintain ABI compatibil
 Versions not listed above are not supported at all.
 
 > [!NOTE]
-> Note that one can enable or disable certain [OpenSSL features] when building it, diverging from the default configuration. The Go runtime does not support all possible configurations, and some may cause the Go runtime to panic during initialization or not work as expected. The Go runtime is tested with the default configuration of the supported versions and with the OpenSSL configuration shipped in [Azure Linux]. 
+> Note that one can enable or disable certain [OpenSSL features] when building it, diverging from the default configuration. The Go runtime does not support all possible configurations, and some may cause the Go runtime to panic during initialization or not work as expected. The Go runtime is tested with the default configuration of the supported versions and with the OpenSSL configuration shipped in [Azure Linux].
```

Comment:
What is 'it' referring to and do we start mention Az Linux 3 SSL impl here and throughout the doc?

Reply:
"It" is OpenSSL. I'm not sure it needs any special mention other than maybe an entry in the changelog to note the Microsoft Go releases that include our compat changes for Azure Linux 3.